client/state/memberships/coupon-list/test/reducer.js

```diff
@@ -0,0 +1,55 @@
+import deepFreeze from 'deep-freeze';
+import {
+	MEMBERSHIPS_COUPONS_RECEIVE,
+	MEMBERSHIPS_COUPON_RECEIVE,
+	MEMBERSHIPS_COUPON_DELETE,
+} from '../../../action-types';
+import { items } from '../reducer';
+
+describe( 'reducer', () => {
+	describe( 'items()', () => {
+		test( 'should default to an empty object', () => {
+			expect( items( undefined, {} ) ).toEqual( {} );
+		} );
+	} );
+
+	describe( 'MEMBERSHIPS_COUPONS_RECEIVE', () => {
+		test( 'should properly add coupons for site', () => {
+			const coupons = [ { ID: 1 }, { ID: 2 } ];
+			const siteId = 1;
+			const expectation = { 1: coupons };
+
+			expect( items( {}, { type: MEMBERSHIPS_COUPONS_RECEIVE, siteId, coupons } ) ).toEqual(
+				expectation
+			);
+		} );
+	} );
+
+	describe( 'MEMBERSHIPS_COUPON_RECEIVE', () => {
+		test( 'should properly add coupons for site', () => {
```

Comment:
Seems like we need an additional test that ensures that coupons stack properly for multiple sites